### ISSUE TITLE
Not working w/ Chrome on date~ input fields

### ISSUE THREAD

**Issue by dows** [xdan/datetimepicker#224]
Datetimepicker is showing but isn't instering anything on `<input type="date" />`, `<input type="datetime" />` and `<input type="datetime-local" />` fields since Chrome shows it's own date pattern and (not so) brilliant datepicker.

**Comment by kemallette** [comment]
TLDR: change input to type="text" and use an input mask to hack around it (not ideal, but it should get you moving)

I've had issues with other datepickers and chrome's 'native' `type="date" fields as well. I've just turn them into normal `type="text"` and use an input mask. I wish there was some way to disable chrome's default on this, but I haven't come across anything to do that, though I haven't really looked.

**Comment by dows** [comment]
I also thought about this solution, but on my project datetimepicker will set globally for all date~ fields. If I set all date~ fields to text, I would have to call datetimepicker for every single field.
I solved it the other way around: I look, if it's Chrome or a new Opera (which is pretty much the same) or a mobile (most of them provide a date picker as well) and just turn datetimepicker off for them. Sure I could make the switch von datetime to text for those, but if the browser serves a good datepicker (as on Android) why not use this?
